### PR TITLE
Portfolio Mobile layout: Added contact session

### PR DESCRIPTION
**Pull request by V0pe**
**Portfolio Mobile layout**

In this part:
- Added the markup for the contact session
- Added the styles for the contact session
- Linked the form to the freespree service

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -196,5 +195,47 @@ <h2>Skills</h2>
...
+                hearing about <br>new projects, so <br>
+                if you'd like to<br> chat please get<br>in touch.
+            </h2>
+        <form action="https://formspree.io/f/mlezlkwz" method="post">
+            <ul>
+            <li><input type="text" name="fullname" placeholder="Full name"
+                maxlength="30" required></li>
+            <li><input type="email" name="email" placeholder="Email address" required></li>
+            <li><textarea name="message" id="form-msg" cols="30" rows="10"
+                placeholder="Enter text here" maxlength="300" required></textarea></li>
```
Comment (TedLivist) [inline index.html]: - [ ] The `maxlength` attribute for your textarea should be **500**, not 300. Kindly fix this.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @V0pe,

Good job so far!
✔️ Good documentation
✔️ Nice design
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (V0pe) [reply index.html]: 👍 Thanks for the observation. I have adjusted accordingly. @TedLivist

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @V0pe,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**